crates/swc_ecma_codegen/tests/fixture/template-literal/output.min.js

```diff
@@ -0,0 +1,15 @@
+const template_literal1=`test${"test"}test${"test"}`;const template_literal2=``;const template_literal3=` `;const template_literal4=`string text`;const template_literal5=`string text line 1
```

Comment:
Weird github behavior on source file, we have good tests on all characters and github things it is binary file

Reply:
Is it possible there's a NUL character at the end of the file in the console.logs? Github marks a file as binary if it finds one of those within the first 8000 bytes: https://github.com/git/git/blob/9c9b961d7eb15fb583a2a812088713a68a85f1c0/xdiff-interface.c#L187-L193

This is what VS Code shows me: 
![image](https://user-images.githubusercontent.com/2777107/159601052-8ed87b54-3272-4a5f-874f-6acc3cf1bcf4.png)

Reply:
I think this is the only concern.
Other things look good to me

Reply:
@Vannevelj @kdy1 Just for tests, in real code it should not be happen

Reply:
Ah, thank you!